setup_workspace.sh

```
@@ -0,0 +1,189 @@
+    ]
+}
+EOF
+
+# Создание файла .gitignore, если его еще нет
+if [ ! -f ".gitignore" ]; then
+    echo -e "${BLUE}Создание файла .gitignore...${NC}"
+    cat > .gitignore << EOF
+# Python
+__pycache__/
+*.py[cod]
+*$py.class
```

Shellcheck (0.10.0)

[warning] py is referenced but not assigned.

(SC2154)